lbry/lbry/extras/daemon/storage.py

```diff
@@ -744,3 +752,20 @@ def get_streams_to_re_reflect(self):
             "where r.timestamp is null or r.timestamp < ?",
             int(self.time_getter()) - 86400
         )
+
+    # # # # # # # # # # dht functions # # # # # # # # # # #
+    async def get_peers(self):
```

Comment:
I think this should be named somewhat more descriptively, like `get_persisted_kademlia_peers`

Reply:
also please add a return type annotation for this